### PR TITLE
Directing 'map/people/only-on-map' to

### PR DESCRIPTION
**Pull request by michaltrzesimiech**
Closes https://github.com/rtcharity/eahub.org/issues/613

### CODE REVIEW

File: eahub/config/urls.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     path('links/', views.LegacyRedirectView.as_view(url='https://resources.eahub.org (temporarily)')),
     path('map/', views.LegacyRedirectView.as_view(pattern_name='profiles')),
     path('map/people/all/', views.LegacyRedirectView.as_view(pattern_name='profiles')),
+    path('map/people/only-on-map', views.LegacyRedirectView.as_view(pattern_name='profiles')),
```
Comment (taymonbeal) [inline eahub/config/urls.py]: Needs a trailing slash.

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: Done. Thanks!

**Comment by taymonbeal** [comment]
Still wondering about above question.

**Comment by michaltrzesimiech** [comment]
Sorry, missed that! I'm not convinced it's useful at all, but I imagine we could change our minds and so the setup in the future?

**Pull request merged**